agent/config/config.go

```diff
@@ -434,6 +445,25 @@ func getTaskCPUMemLimitEnabled() Conditional {
 	return taskCPUMemLimitEnabled
 }
 
+func getTaskMetatadataRpsLimits() (int, int) {
+	var rpsLimit, rpsBurstLimit int
+	rpsLimitEnvVal := os.Getenv("ECS_TASK_METADATA_RPS_LIMIT")
+	rpsLimitSplits := strings.Split(rpsLimitEnvVal, ",")
+	if len(rpsLimitSplits) != 2 {
+		seelog.Warnf("Invalid format for \"ECS_TASK_METADATA_RPS_LIMIT\", expected: \"rateLimit,burst\"")
+	} else {
```

Comment:
just `return` here so that you don't need an additional `else` block. Avoids one level of indentation

Reply:
This block got moved into this method from the original `environmentConfig()` and was never refactored. Will make this change.